saltproc/saltproc.py

```diff
@@ -392,8 +402,11 @@ def reopen_db(self, restart):
 
             # write new material file
             self.core = {}
-            self.core[self.driver_mat_name] = self.driver_after_db[self.current_step - 1]
-            self.core[self.blanket_mat_name] = self.blanket_after_db[self.current_step - 1]
+            self.core[self.driver_mat_name] = self.driver_after_db[self.current_step - 2]
+            self.core[self.blanket_mat_name] = self.blanket_after_db[self.current_step - 2]
+            print(self.current_step - 1)
+            print(self.core[self.driver_mat_name])
+            print(self.core[self.blanket_mat_name])
```

Comment:
Are these print statements for debugging or runtime logging? If the former, I recommend pdb. If the latter, I recommend adding a logging system and .log file output to saltproc.